source/scripting/Screen.cs

```diff
@@ -196,28 +196,30 @@ public static class Screen
 		/// The base width of the screen used for all UI Calculations, unless ScaledDraw is used
 		/// </summary>
 		public const float Width = 1280f;
-
 		/// <summary>
 		/// The base height of the screen used for all UI Calculations
 		/// </summary>
 		public const float Height = 720f;
-
 		/// <summary>
 		/// Gets the current Screen Aspect Ratio
 		/// </summary>		   
 		public static float AspectRatio
 		{
-			get { return Function.Call<float>(Hash._GET_SCREEN_ASPECT_RATIO, 0); }
+			get
+			{
+				return Function.Call<float>(Hash._GET_SCREEN_ASPECT_RATIO, 0);
+			}
 		}
-
 		/// <summary>
 		/// Gets the width of the scaled against a 720pixel height base.
 		/// </summary>
 		public static float ScaledWidth
 		{
-			get { return Height*AspectRatio; }
```

Comment:
There was nothing wrong with this, if it fits in a single line, that's good! @crosire?